build-image-src/Dockerfile-go1x

```
@@ -25,7 +25,7 @@ RUN yum groupinstall -y development && \
   && yum clean all
 
 # Install Go
-RUN curl -L https://go.dev/dl/go1.20.linux-amd64.tar.gz | tar -zx -C /usr/local
+RUN curl -L https://go.dev/dl/go1.21.6.linux-amd64.tar.gz | tar -zx -C /usr/local
```

nit: we could fetch latest Go version too

```suggestion
RUN curl -L https://go.dev/dl/$(curl -L "https://go.dev/VERSION/?m=text" | grep go1.).linux-amd64.tar.gz | tar -zx -C /usr/local
```